src/useEditor.ts

```diff
@@ -111,15 +111,17 @@ export const useEditor = (externalProps: ExternalProps) => {
           trackState.status !== TrackChangesStatus.viewSnapshots) &&
         !props.isComparingMode
       ) {
-        const sendable = sendableSteps(nextState)
-
-        if (sendable) {
-          collabProvider.sendSteps(
-            sendable.version,
-            sendable.steps,
-            sendable.clientID,
-            false
-          )
+        const skipSync = tr.getMeta('skipCollabSync')
+        if (!skipSync) {
```

Comment:
sync skipping will not do much because any next transaction will cause a comparing and resync with backend and will send all the missing difference to it anyway to resync. If it will unable to resync the doc will never be saved. We cant really modify the way backend and frontend communicates to support something, what is essentially a drafting mode for comments.